qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy

```diff
@@ -142,7 +141,7 @@ class AdmissionControllerTest extends BaseSpecification {
 
         assert ClusterService.updateAdmissionController(ac)
         // Maximum time to wait for propagation to sensor
-        Helpers.sleepWithRetryBackoff(5000)
+        sleep(5000)
```

Comment:
I think the goal of `sleepWithRetryBackoff` was to sleep longer on each test retry?
Perhaps we need to evaluate whether getting rid of this is not going to increase the flake rate significantly.

Food for thought: it would be very interesting to see which tests are being retried, how many times and how often. Another use case for the test accounting epic?